test/functional/feature_init.py

```diff
@@ -137,8 +138,8 @@ def check_clean_start():
                     # Since the genesis block is not checked by -checkblocks, the
                     # perturbation window must be chosen such that a higher block
                     # in blk*.dat is affected.
-                    tf.seek(150)
-                    tf.write(b'1' * 200)
+                    tf.seek(randint (150, 15000))
```

Comment:
In 5ab6419f380cc0a8cde78b125f3eeee5fcba43ae: What is the rationale of the range 150~15000?

Reply:
I tried to pick a range large enough for randomization to be efficient, but the range itself is arbitrary and could be adjusted.

Reply:
For more clarification @brunoerg the 150 starting point was chosen in the previous test to exclude the genesis block iiuc. The 15000 range should be large enough to provide efficient randomization but small enough to not end outside of the blockfile (200 blocks in the test with ~160 characters each). Could also shorten or enlarge the range ((200*160 = 32000 characters) but I thought 15000 would a good middle ground. 

Let me know if you have any suggestions. Can also add a line of documentation if necessary.